moped-database/metadata/tables.yaml

```diff
@@ -3399,6 +3253,245 @@
+          - status_id
+          - status_note
+  select_permissions:
+    - role: moped-admin
+      permission:
+        columns:
+          - contract_amount
+          - contract_number
+          - contractor
+          - created_at
+          - created_by_user_id
+          - description
+          - id
+          - implementation_workgroup_id
+          - interim_work_order_id_old
+          - is_deleted
+          - project_id
+          - status_id
+          - status_note
+          - task_orders
+          - updated_at
+          - updated_by_user_id
+          - work_assignment_id
+        filter: {}
+    - role: moped-editor
+      permission:
+        columns:
+          - is_deleted
+          - contract_amount
+          - created_by_user_id
+          - id
+          - project_id
+          - status_id
+          - updated_by_user_id
+          - task_orders
+          - contract_number
+          - contractor
+          - description
+          - implementation_workgroup_id
+          - interim_work_order_id_old
+          - status_note
+          - work_assignment_id
+          - created_at
+          - updated_at
+        filter: {}
+    - role: moped-viewer
+      permission:
+        columns:
+          - is_deleted
+          - contract_amount
+          - created_by_user_id
+          - id
+          - project_id
+          - status_id
+          - updated_by_user_id
+          - task_orders
+          - contract_number
+          - contractor
+          - description
+          - implementation_workgroup_id
+          - interim_work_order_id_old
+          - status_note
+          - work_assignment_id
+          - created_at
+          - updated_at
+        filter: {}
+  update_permissions:
+    - role: moped-admin
+      permission:
+        columns:
+          - contractor
+          - contract_number
+          - description
+          - project_id
+          - is_deleted
+          - work_assignment_id
+          - contract_amount
+          - interim_work_order_id_old
+          - implementation_workgroup_id
+          - task_orders
+          - status_id
+          - status_note
+        filter: {}
+        check: {}
+        set:
+          updated_by_user_id: x-hasura-user-db-id
+    - role: moped-editor
+      permission:
+        columns:
+          - contractor
+          - contract_number
+          - description
+          - project_id
+          - is_deleted
+          - work_assignment_id
+          - contract_amount
+          - interim_work_order_id_old
+          - implementation_workgroup_id
+          - task_orders
+          - status_id
+          - status_note
+        filter: {}
+        check: {}
+        set:
+          updated_by_user_id: x-hasura-user-db-id
+  event_triggers:
+    - name: activity_log_moped_proj_work_activity
+      definition:
+        delete:
+          columns: '*'
+        enable_manual: false
+        insert:
+          columns: '*'
+        update:
+          columns: '*'
```

Comment:
i'll open a separate issue to make sure we handle these new columns in the activity log helpers.

Reply:
https://github.com/cityofaustin/atd-data-tech/issues/13000